src/main/java/org/myrobotlab/service/MouthControl.java

```diff
@@ -23,27 +29,72 @@
 
   private static final long serialVersionUID = 1L;
   public final static Logger log = LoggerFactory.getLogger(MouthControl.class);
+  @Deprecated /* future releases members will be protected use appropriate methods instead */
   public int mouthClosedPos = 20;
+  @Deprecated /* future releases members will be protected use appropriate methods instead */
   public int mouthOpenedPos = 4;
+  @Deprecated /* future releases members will be protected use appropriate methods instead */
   public int delaytime = 75;
+  @Deprecated /* future releases members will be protected use appropriate methods instead */
   public int delaytimestop = 150;
+  @Deprecated /* future releases members will be protected use appropriate methods instead */
   public int delaytimeletter = 45;
+  
+  protected final Set<String> servoServices = new HashSet<>();

```

Comment:
where ever we need the list of servo services, we should just ask runtime for it... same with speechServices below.

Reply:
The list changes - it can't just be based on a service querying runtime ... which should be possible.
But if the registry changes - and event needs to be communicated for "interested" services

Reply:
I agree - removed
Processing is done with normalized runtime data